src/components/Board/Output/Output.container.js

```diff
@@ -184,8 +184,14 @@ export class OutputContainer extends Component {
   handleCopyClick = () => {
     const { intl, showNotification } = this.props;
     const labels = this.props.output.map(symbol => symbol.label);
-    navigator.clipboard.writeText(labels.join(' '));
-    showNotification(intl.formatMessage(messages.copyMessage));
+    try {
+      navigator.clipboard.writeText(labels.join(' '));
+      showNotification(intl.formatMessage(messages.copyMessage));
+    } catch (err) {
+      showNotification(
+        `${intl.formatMessage(messages.failedToCopy)}: ${err.message}`
```

Comment:
exception `error.message ` should not be displayed to the user. Log it to the console.

Reply:
Is it okay now? Please tell us if we have to correct anything else

Reply:
if you catch an error, you should showNotification 'failed to copy' to the user.  And only console log `err.message`. Is a message for dev.